lib/k8s/util.rb

```diff
@@ -3,6 +3,23 @@
 module K8s
   # Miscellaneous helpers
   module Util
+    module ExceptionlessBangMethod
+      private
+
+      # @!macro [attach] exceptionless_bang_method
```

Comment:
This documents the dynamically generated `bang!` methods, but I couldn't find any way to replicate the param list / return value, so the bang-variants just show `(*args, **options) => Object`  in the final doc.